assets/css/style.css

```diff
@@ -916,20 +986,35 @@ color:white !important;
     }
 }
 
+.word-wrap {
+    word-wrap: break-word;
+}
+
 .text-body {
     padding: 100px;
-    font-size:40px;
+    padding-top: 0px;
+    font-size:35px;
     text-align: center;
     font-family: 'Open Sans', sans-serif;
+    word-wrap: break-word;
+
+}
+
+.text-body-20px {
+    padding: 100px;
+    font-size:20px;
+    font-family: 'Open Sans', sans-serif;

```

Comment:
It seems like this isn’t used anywhere? 🤔

Reply:
Yeah, this isn't used. Removing